x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elasticsearch/CustomElandModel.java

```diff
@@ -39,31 +34,10 @@ public CustomElandModel(
     }
 
     @Override
-    public ActionListener<CreateTrainedModelAssignmentAction.Response> getCreateTrainedModelAssignmentActionListener(
```

Comment:
Moved this common logic into the base class and added the `String modelNotFoundErrorMessage()` function to handle the difference